src/Core/src/Handlers/MenuFlyoutItem/MenuFlyoutItemHandler.Windows.cs

```diff
@@ -37,6 +37,11 @@ public static void MapText(IMenuFlyoutItemHandler handler, IMenuFlyoutItem view)
 			handler.PlatformView.Text = view.Text;
 		}
 
+		/// <summary>
+		///	Maps accelerator to the Windows platform control.
+		/// </summary>
+		/// <param name="handler">The handler, of type IMenuFlyoutItemHandler.</param>
```

Comment:
nit: you can add the cref tag to create automatic links when the documentation is autmatically made :) 
```suggestion
/// <param name="handler">The handler, of type <see cref="IMenuFlyoutItemHandler">.</param>
```

Reply:
Hmmm I don't know why GH didn't pick up the whitespace on the suggestion :/